l10n_br_fiscal/models/payment_mixin.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2020 KMEE
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from ..constants.payment import (
+    FORMA_PAGAMENTO,
+)
+
+
+class PaymentMixin(models.AbstractModel):
+    _name = "l10n_br_fiscal.payment.mixin"
+    _description = "Fiscal Payment Mixin"
+
+    _payment_inverse_name = 'document_id'  # Modify on your class
+
+    def _compute_payment_change_value(self):
+        """
+            @api.depends("amount_total", "fiscal_payment_ids")
+            def _compute_payment_change_value(self):
+                self.___compute_payment_change_value()
+
+        :return:
+        """
+        raise NotImplementedError
+
+    def _date_field(self):
+        return self.document_date
+
+    def _get_amount_total(self):
+        return self.amount_financial
```

Comment:
```suggestion
        return self.amount_financial_total
```
o nome dessa variavel foi refatorado no merge dessa pr:

https://github.com/OCA/l10n-brazil/pull/1502